### PR TITLE
Handle properly configs with invalid Liquid templates

### LINKED ISSUE #909: Some liquid functions are not working - especially the time functions
All of the time functions are returning the empty string:

**Functions:** `time`, `today`, `http_time`, `cookie_time`, `localtime`, `now`, `utctime`

##### Version

3scale 2.3-CR1

##### Steps To Reproduce

1. Create application
2. add URL rewrite policy
3. add rules for any of these functions with liquid
  Example: `arg` = `{{ now }}`
4. Make a request and response should look like:
```
"args": "user_key=5941428bd561fa5a91489f2c36ed8cb3&arg=",
```

##### Current Result
```json
{
  "method": "GET",
  "path": "/test/path",
  "args": "user_key=5941428bd561fa5a91489f2c36ed8cb3&arg=",
  "body": "",
...
}
```
##### Expected Result
```json
{
  "method": "GET",
  "path": "/test/path",
  "args": "user_key=5941428bd561fa5a91489f2c36ed8cb3&arg=TIMESTAMP",
  "body": "",
...
}
```

### PR DESCRIPTION
**Pull request by davidor**
Closes partially #908

### CODE REVIEW

**Comment by davidor** [comment]
You are right, it's #908

**Comment by davidor** [comment]
And as you mentioned we should probably make this generic and not only liquid-specific.

**Comment by mikz** [comment]
@davidor yes, something along the lines of https://github.com/3scale/apicast/pull/854

That one, unfortunately, missed the failure in `.new`.

File: gateway/src/apicast/policy_chain.lua [hunk 1]
```diff
@@ -87,7 +88,12 @@ function _M.load_policy(module, version, ...)
         local mod, err = policy_loader:pcall(module, version or 'builtin')
 
         if mod then
-            return mod.new(...)
+            local new_policy_ok, res = pcall(mod.new, ...)
```
Comment (mikz) [inline gateway/src/apicast/policy_chain.lua]: It might be good to expect a second return value `err`. It is possible a policy could `return nil, 'error'`.

Comment (davidor) [reply gateway/src/apicast/policy_chain.lua]: Fixed :+1:

**Pull request merged**